.circleci/config.yml

```diff
@@ -1177,21 +1200,33 @@ workflows:
           requires:
             - dependencies_bundler
             - dependencies_npm
+      - licenses:
+          executor: builder-ruby26
+          requires:
+            - dependencies_bundler
+      - swagger:
+          executor: builder-ruby26
+          requires:
+            - dependencies_bundler
       - lint:
           executor: builder-ruby26
           requires:
-            - assets_precompile
+            - dependencies_npm
       - jest:
           executor: builder-ruby26
           requires:
             - dependencies_npm
       - notify_success:
           requires:
+            - licenses
+            - swagger
             - lint
             - jest
           <<: *only-master-filter
       - notify_failure:
           requires:
+            - licenses
+            - swagger
```

Comment:
leave off `notify_failure`. @jlledom is fixing some stuff with that so better avoid conflict.

Reply:
Looks good otherwise and no additional approval from me needed.